config/peerpods/podvm/gcp-podvm-image-cm.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: gcp-podvm-image-cm
+  namespace: openshift-sandboxed-containers-operator
+data:
+  # PodVM image distro
+  PODVM_DISTRO: rhel
+
+  # Image
+  GCP_IMAGE_BASE_NAME: "podvm-image"
+  GCP_IMAGE_VERSION_MAJ_MIN: "0.0"
+
+  # Packer Instance type
+  MACHINE_TYPE: "e2-medium"
+
+  # Pod VM sources
+  # If changing the source, then ensure the respective payload binaries are available
+  # for the new source
+  CAA_SRC: "https://github.com/confidential-containers/cloud-api-adaptor"
+  CAA_REF: "main"
+
+  # Booleans
+  INSTALL_PACKAGES: "no"
+  DOWNLOAD_SOURCES: "no"
+  CONFIDENTIAL_COMPUTE_ENABLED: "no"
+  DISABLE_CLOUD_CONFIG: "true"
+  ENABLE_NVIDIA_GPU: "no"
+  UPDATE_PEERPODS_CM: "yes"
+  BOOT_FIPS: "no"
+
+  # NVIDIA GPU vars
+  NVIDIA_DRIVER_VERSION: "535"
+  NVIDIA_USERSPACE_VERSION: "1.13.5-1"
```

Comment:
I'd remove vars which are not actually used by GCP bootc image creation

Reply:
@snir911 isn't this generic to the `prepare_source_code` (`lib.sh`) used by the "scratch" method? I mean, for now yes, we would be using "from prebuilt" /bootc method here. But maybe just keep it commented with some notes saying this is only used by the "from scratch" method?

Also, for my understanding... will "from prebuilt" have those nvidia customization?

Reply:
sorry, missed that comment, this will most likely be dropped once bootc will be the default building method, hence, I'd already drop it for gcp